README.md

```diff
@@ -1,20 +1,30 @@
 # Standalone, Android and MacOS Build with Java Tests
 
-This repository shows a few Java tests that use the page object model and AltUnityTester to test the Unity endless runner sample:
+This repository shows a few Java tests that use the page object model and AltTester to test the Unity endless runner sample:
 https://assetstore.unity.com/packages/essentials/tutorial-projects/endless-runner-sample-game-87901
 
 ### Running the tests on Windows or MacOS
-The tests are meant to be run on an Windows or MacOS device. The app is provided in this repository, under the mac/  and windows/ folder.
+The tests are meant to be run on an Windows or MacOS device. 
 
 To start the tests, depending of your OS run:
+
 - ./start_tests_mac.sh on MacOS/Linux
+
+    Create a folder `mac` under project.
+    The app is provided at https://altom.com/app/uploads/AltTester/TrashCat/TrashCatMacOS.zip and needs to be included under the mac/ folder.
```

Comment:
Change the Mac download link to https://altom.com/app/uploads/AltTester/TrashCat/TrashCatMacOS.app.zip since the zip file was changed